lib/tasks/mailers.rake

```diff
@@ -17,13 +17,29 @@ namespace :mailers do
     user = User.new(email: 'foobar@foobar.foo', password: 'foobarfoobar2')
     test_patient = Patient.new(creator: user)
     test_patient.responder = test_patient
+    test_patient.first_name = "Test"
+    test_patient.last_name = "McTest"
     test_patient.email = '<test_email>'
     test_patient.submission_token = SecureRandom.urlsafe_base64[0, 10]
     test_patient.jurisdiction = Jurisdiction.where(name: 'County 1').first
+    test_patient.primary_language = 'eng'
     test_patient.save!
     PatientMailer.enrollment_email(test_patient).deliver_now
   end
 
+  desc "Test making an enrollment sms"
+  task test_enrollment_sms: :environment do
+    # patient = Patient.first.dup
```

Comment:
is there a reason this is commented out?

Reply:
The other Twilio rake tasks are commented out. I kind of like that it requires uncommenting, since the Twilio-based tasks should not be run unless the developer really knows what they're doing (not sure if that was the rationale for commenting out the other ones, but perhaps).